inc/lock.class.php

```diff
@@ -980,6 +985,21 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
    static function isFieldLocked($a_lockable, $field) {
       return in_array($field, $a_lockable);
    }
+
+   static function showLocksForAnItem(CommonDBTM $item) {
+      $pflock = new self();
```

Comment:
in this function, a title for the tab should permits to disctinct locks from glpi